Examples/KerrBH/params.txt

```diff
@@ -60,8 +60,8 @@ max_level = 6 # There are (max_level+1) grids, so min is zero
 # Need one for each level except the top one, ie max_level items
 # Generally you do not need to regrid frequently on every level
 # Level Regridding:   0   1   2   3   4   5
-regrid_interval   =  50  25   5   5   5   5
-regrid_thresholds = 0.1 0.1 0.1 0.1 0.1 0.1
+regrid_interval   =   0   0   0   0   0   0
```

Comment:
I would leave the regrid threshold as it was at 0.1 so the example is not too heavy, unless it is needed? The regrid interval could just be 1 0 0 0 0 0, so that some regridding does happen for the change in gauge.